apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -76,9 +76,13 @@ export class BulkScriptAction extends BaseScriptAction {
   }
 
   public run(): Promise<void> {
-    if (!FileSystem.exists(this.rushConfiguration.rushLinkJsonFilename)) {
+    const lastInstallFlag: LastInstallFlag = new LastInstallFlag(
+      this.rushConfiguration.commonTempFolder,
+      LastInstallFlag.getCurrentState(this.rushConfiguration)
```

Comment:
Maybe we should just pass `this.rushConfiguration` to the `LastInstallFlag` constructor and do this all in the constructor.

Reply:
The path for the flag can't be assumed to be the `commonTempFolder` as there are other uses for it, so at least that parameter needs to be kept. For the second, we would need to mock out some parameters (`process.versions.node`, `RushConfiguration` params, etc.)... I just am not sure how that's done :) I'll reach out offline to ask questions on this